apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.IntegrationTests/JobsTest.cs

```diff
@@ -98,7 +100,7 @@ public void ListJobs_JobLabels()
 
             // Find the job after listing with full projection.
             var options = new ListJobsOptions { Projection = ProjectionEnum.Full };
-            var jobFound = client.ListJobs(options).Single(job => job.Reference.JobId == jobToFind.Reference.JobId);
+            var jobFound = client.ListJobs(options).FirstOrDefault(job => job.Reference.JobId == jobToFind.Reference.JobId);
 
             VerifyJobLabels(jobFound.Resource.Configuration.Labels);
```

Comment:
Maybe add here `Asser.NotNull(jobFound)`, its better than to get the `NullReferenceException`.

Reply:
Sure, will do.